raml-parser-2/src/test/resources/org/raml/v2/parser/examples/builtin/date-validation/input.raml

```diff
@@ -7,6 +7,8 @@ types:
       good: 2015-05-23
       bad: 2016-13-33
       bad-2: 2016-12-33
+      bad-3: 16-12-30
+      bad-4: 20191-12-30
```

Comment:
What's the idea for this test? I see you added two invalid dates, but this change does not make the test fail